bucket/mailsend.json

```
@@ -1,19 +1,18 @@
 {
+    "homepage": "https://github.com/muquit/mailsend",
     "version": "1.19",
+    "description": "A program to send mail via SMTP from command line.",
     "license": "BSD-3-Clause",
     "url": "https://github.com/muquit/mailsend/releases/download/1.19/mailsend1.19.exe.zip",
-    "homepage": "https://github.com/muquit/mailsend",
-    "checkver": {
-        "github": "https://github.com/muquit/mailsend"
-    },
+    "hash": "sha1:9f25e39200089d8c281949323eadb7ab18352dff",
+    "pre_install": "Rename-Item \"$dir\\mailsend$version.exe\" \"$dir\\mailsend.exe\"",
+    "bin": "mailsend.exe",
+    "checkver": "github",
     "autoupdate": {
-        "url": "https://github.com/muquit/mailsend/releases/download/$version/mailsend$version.exe.zip"
-    },
-    "hash": "87db23b8c9588353d12e19e953dddb17ed04b422f41bcfbaa8ad4e27057245b4",
-    "bin": [
-        [
-            "mailsend1.19.exe",
-            "mailsend"
-        ]
-    ]
+        "url": "https://github.com/muquit/mailsend/releases/download/$version/mailsend$version.exe.zip",
+        "hash": {
+            "url": "https://github.com/muquit/mailsend/releases/latest",
```

This will make impossible to use older version installing via `mailsend@whatever`. It will find only latest hash not version specific.
Also notes about deprication could be used.
```suggestion
            "url": "https://github.com/muquit/mailsend/releases/tag/$version",
```